src/login.rs

```diff
@@ -203,6 +207,9 @@ impl Login {
     fn previous(&self) {
         let self_ = imp::Login::from_instance(self);
         self_.content.set_visible_child_name("phone-number-page");
+
+        // grab focus for entry after reset
```

Comment:
```suggestion
        // Grab focus for entry after reset
```

Let's keep the comment capitalization coherent. Also with the other comments below.